mackup/applications/slack.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = Slack
+
+[configuration_files]
+Library/Application Support/Slack
```

Comment:
I can see lots of local specific stuff in this folder, why syncing it?
<img width="1157" alt="screen shot 2018-10-29 at 1 28 32 pm" src="https://user-images.githubusercontent.com/122286/47678276-9c555380-db7e-11e8-8947-68dac9cf6d06.png">

Reply:
Caches can be ignored. It's because syncing signed-in slack accounts.

Reply:
Do you know how Slack keeps authorization credentials stored?

Just off intuition, this feels like a security risk, in the event someone decides to put this as part of a public repository. (I haven't looked into this at all.)

Also, this doesn't support Linux. 😕 (As far as I understand, `mackup`, is attempting to build for both? @lra)

Reply:
I'm not worried about the security, if people make their backup public I can't help them.

Re. GNU/Linux, that's true, but lots of applications only support one OS and support the other later, that's not a blocker.

The blocker here is that we should not sync non-configuration stuff, or cache.

Reply:
So for Linux, I meant that the path listed seems to only be the MacOS location, not a Linux location. (I don't have a present GUI install of Linux so I can't provide much feedback there.)

As for security, sure – but I definitely think there should be warnings if there aren't already. (I've never seen such warnings, so I'm not sure they exist.)

Reply:
There was a warning for private keys, it was ignored anyway. There is now an explanation of what it does https://github.com/lra/mackup/blob/master/mackup/main.py#L30-L34

But really, you have to explicitly configure the backup method to use (or it uses Dropbox, which is private by default) and you have to explicitly state the application you want to backup.

Re. Paths, I was talking about the same thing, applications supported in Mackup generally only start with the MacOS path, and when someone using Linux shows up, it adds the Linux path later, no harm in this.